server/controllers/article.controller.js

```diff
@@ -149,22 +149,31 @@ class ArticleController {
      * @returns {object}
      * @memberof getAllArticles
     */
+    const { page } = req.params;
+    const limit = 10;
+    const offset = limit * (page - 1);
+    const order = [['updatedAt', 'DESC']];
     const articles = await Article.findAndCountAll({
+      limit,
+      offset,
+      order,
       where: { published: true },
       include: [{
         model: User,
         as: 'author',
         attributes: ['firstname', 'lastname', 'bio', 'image']
-      },
-      {
-        model: Comment,
-        as: 'comments'
       }]
     });
+
     if (articles.count === 0) {
       return Response(res, 400, 'There is no article in database');
     }
-    return Response(res, 200, 'Articles successfully retrieved', { articles });
+    const pages = Math.ceil(articles.count / limit);
+
+    if (page > pages) {
+      return Response(res, 404, 'There are no articles here');
+    }
+    return Response(res, 200, 'Articles successfully retrieved', { articles, pages });
   }
```

Comment:
Nice initiative on catering for the different edge cases in terms of the page number. However, we need extra information in the response to let us know if the current articles being displayed are the last set of articles, or there are more articles to come. That way, we can easily know when to display a 'Load More' button to the user on the frontend. Please look into this.

Reply:
Okay, will look into this.